### PR TITLE
Squashing cockroaches now makes a mess

### LINKED ISSUE #22951: Squashing cockroaches don't make cockroach gibs anymore
:(

### PR DESCRIPTION
**Pull request by coiax**
Fixes #22951.

:cl: coiax
bugfix: Squashing cockroaches now leaves a mess.
/:cl:

### CODE REVIEW

File: code/modules/mob/living/simple_animal/friendly/cockroach.dm [hunk 1]
```diff
@@ -31,8 +31,8 @@
 	..()
 
 /mob/living/simple_animal/cockroach/proc/squish()
-	adjustBruteLoss(1) // kills a normal cockroach
 	new /obj/effect/decal/cleanable/deadcockroach(get_turf(src))
```
Comment (ChangelingRain) [inline code/modules/mob/living/simple_animal/friendly/cockroach.dm]: but what if the cockroach has more than 1 health?

Comment (coiax) [inline code/modules/mob/living/simple_animal/friendly/cockroach.dm]: That's intentional. I considered the case of a human-level intelligence cockroach that has 200 health. It seems unfair to just get suddenly crushed, so if a cockroach gets more than 1 health, it survives fine. See https://youtu.be/L7W2HKtdWRo

**Comment by Core0verload** [comment]
Removing cockroaches would be a better fix.

File: code/modules/mob/living/simple_animal/friendly/cockroach.dm (hunk 1 above)
Comment (RemieRichards) [inline code/modules/mob/living/simple_animal/friendly/cockroach.dm]: but they'll poop out gibs everytime they're stepped on?
200 HP is 200 gibs.

Comment (coiax) [inline code/modules/mob/living/simple_animal/friendly/cockroach.dm]: Well don't step on them then

**Review by ChangelingRain — CHANGES_REQUESTED** [review]
>Well don't step on them then

This is not acceptable reasoning please fix your shit

**Comment by coiax** [comment]
I'm confused, stepping on a sentient cockroach is rude anyway, and it's a rare event?

This is not a janitor friendly codebase

File: code/modules/mob/living/simple_animal/friendly/cockroach.dm (hunk 1 above)
Comment (optimumtact) [inline code/modules/mob/living/simple_animal/friendly/cockroach.dm]: Come on, make it only gib when they actually die, this is lazy work and only I get to do that

**Comment by ChangelingRain** [comment]
All you had to do was fix the issue! #23117

**Pull request closed without merging**